lib/solidus_bactracs/api/shipment_serializer.rb

```diff
@@ -107,9 +104,8 @@ def safe_rma_type
 
       def get_rp_location
         #Double verifing
-        return nil if rma_type == "4"
+        return nil if safe_rma_type == "4"
 
-        rp_location = @shipment.get_rma_rp_location if @shipment.get_rma_rp_location
```

Comment:
this didn't seem to be used at all.  Also, it directly called the object properties, rather than relying on the config proc as the 'interface' for this data